{"name": "User login with incorrect credentials", "status": "passed", "attachments": [{"name": "stdout", "source": "85ce12b2-4e47-4d06-8300-cdd59b4a5b85-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "payload", "value": "{'email': 'yozhik_ya@yandex.ru'}"}], "start": 1732029584368, "stop": 1732029584672, "uuid": "0efed605-8169-4649-bb27-661cf2890bfa", "historyId": "b786912d1e07a8d7b28dd5a8cf899848", "testCaseId": "76148a48b7585b4a8f8a20e34c40f04e", "fullName": "tests.user.test_user_login.TestUserLogin#test_user_login_incorrect_credentials", "labels": [{"name": "suite", "value": "User Tests"}, {"name": "parentSuite", "value": "API Tests"}, {"name": "subSuite", "value": "User Login Tests"}, {"name": "host", "value": "ubuntu22"}, {"name": "thread", "value": "10772-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.user.test_user_login"}]}